app/controllers/organizations/staff/matches_controller.rb

```diff
@@ -32,7 +32,7 @@ def destroy
   private
 
   def match_params
-    params.require(:match).permit(:pet_id, :adopter_foster_account_id, :match_type, :start_date, :end_date)
+    params.require(:match).permit(:pet_id, :match_type, :start_date, :end_date)
```

Comment:
Does this need a `:person_id` ?

Reply:
Added 85edebb507fd

Reply:
This commit also fixes creating adoptions